rllab/algos/batch_polopt.py

```diff
@@ -4,6 +4,7 @@
 import rllab.misc.logger as logger
 import rllab.plotter as plotter
 from rllab.policies.base import Policy
+from rllab.sampler.utils import rollout
```

Comment:
PEP8: alphabetical imports

Reply:
Can you explain which part isn't in alphabetical order? I'm not sure why.